website/docs/sdk-reference/python.md

```diff
@@ -593,6 +593,11 @@ client = configcatclient.get('#YOUR-SDK-KEY#',
 )
 ```
 
+:::info
+The Python SDK supports [shared caching](/docs/advanced/caching/#shared-cache) since v8.0.0. As for other SDKs, please check their docs for the minimum versions
```

Comment:
```suggestion
The Python SDK supports [shared caching](/docs/advanced/caching/#shared-cache) since v8.0.0. As for other SDKs, check their docs for minimum versions required for shared caching. 
```